src/pages/help-centre/components/_translation-components.tsx

```diff
@@ -19,8 +19,19 @@ const Link = styled(LocalizedLink)`
     }
 `
 
-const getComponent = (key: number, to: string, type: 'link' | 'strong') => {
+const getComponent = (key: number, to: string, type: 'link' | 'deriv_app_link' | 'strong') => {
```

Comment:
To make the deriv app link localized and redirect to the respective language for the link I needed to add new value to pass the type to it.